Makefile

```diff
@@ -2,7 +2,7 @@
 NAMESPACE=keycloak
 PROJECT=keycloak-operator
 PKG=github.com/keycloak/keycloak-operator
-OPERATOR_SDK_VERSION=v0.12.1
+OPERATOR_SDK_VERSION=v0.15.2
```

Comment:
This actually should be `v0.15.1`, see https://github.com/keycloak/keycloak-operator/blob/master/go.mod#L9

Reply:
Updated to 0.15.1